docs/cni/rkt/VagrantCoreOS.md

```diff
@@ -0,0 +1,226 @@
+
+## 3. Create the networks
+
+You can configure multiple networks when using rkt and each network is represented by a configuration file in 
+/etc/rkt/net.d/. Connections to a given container are only allowed from containers on the same network. 
+Containers on multiple networks can be accessed by containers on each network that it is connected to.
+
+To define a rkt network for Calico, create a configuration file in /etc/rkt/net.d/.
+- Each network should be given a unique "name"
+- To use Calico networking, specify "type": "calico"
+- To use Calico IPAM, specify "type": "calico-ipam" in the "ipam" section.
+
+This worked example creates two rkt networks. Run these commands on both `calico-01` and `calico-02`
+```
+cat >/etc/rkt/net.d/10-calico-net1.conf <<EOF
+{
+    "name": "net1",
+    "type": "calico",
+    "ipam": {
+        "type": "calico-ipam"
+    }
+}
+EOF
+
+cat >/etc/rkt/net.d/10-calico-net2.conf <<EOF
+{
+    "name": "net2",
+    "type": "calico",
+    "ipam": {
+        "type": "calico-ipam"
+    }
+}
+EOF
+```
+
+
+## 4. Create a webserver on one of the networks
+
+With the networks created, let's start a container on `calico-01`
+
+### On calico-01
+Create a webserver using an `nginx` container. We'll put this on `net1` network.
+
+Because `rkt` doesn't support daemonizing containers directly, we'll use [systemd-run][systemd-run]
+to create a transient systemd service.
+
+    sudo systemd-run rkt run --store-only=true --insecure-options=image --net=net1 --mount volume=myvol,target=/var/run --volume=myvol,kind=empty docker://nginx
+ 
+ This will produce output like
+
+    Running as unit run-16961.service.
+    
+ Normal [systemd commands][systemd-run] can then be used to get the status of the container and view its logs. 
+ 
+ We can then use `rkt list` to show the IP address
+  
+    $ sudo rkt list
+	UUID            APP     IMAGE NAME                                      STATE   NETWORKS
+	592b9eec        nginx   registry-1.docker.io/library/nginx:latest       running net1:ip4=192.168.0.0, default-restricted:ip4=172.16.28.2
+
+ 
+ So we can see that we now have an `nginx` container running on the network `net1` with an IP address of `192.168.0.0`
+ rkt also creates a second network called `default-restricted`. This is used for communication with the rkt metadata service running on the host and is covered in the [rkt documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md#the-default-restricted-network)
+ 
+## 5. Validate access to webserver
+
+Now that we've created a webserver and we know it's IP address, we can access it using `wget` from containers running on 
+either host, as long as they are created on the same network.
+
+On calico-02
+
+    sudo rkt run --net=net1 --store-only=true --insecure-options=image docker://busybox --exec=/bin/wget -- -T 1 -q -O - 192.168.0.0 2>/dev/null
```

Comment:
Do we actually want to send stderr to null? It might be useful to show rkt starting the pod.